bin/cloudgov/events/get-events

```diff
@@ -65,14 +65,21 @@ SPACE_GUID=$(cf space $SPACE --guid)
 cf target -s $SPACE &>/dev/null
 
 declare -A SPACE_GUIDS=()
-# Keep the code needed to query multiple spaces for events. Not working now, but may be possible.
-#  for SPACE in dev stage prod shared-egress shared-egress-dr tools; do
-for SPC in $SPACE; do
+
+# Attempt to fetch uuids for all usagov cf spaces:
+SPACE_QUERY=''
+for SPC in dr dev stage prod shared-egress tools; do
+#for SPC in $SPACE; do
   if [ ! cf space $SPC &> /dev/null ]; then
     echo "ERROR: CF space '$SPC' does not exist!"
```

Comment:
Just FYI, I saw this message a few times while we were re-deploying the egress proxies tonight. Of course the real problem was that the `cf` call failed because the egress was blocked.